nep-dapi.mediawiki

```diff
@@ -797,6 +797,90 @@ Example
 </pre>
 
 
+====verifyMessage====
+Returns whether the provided signature data matches the provided message and was signed by the account of the provided public key.
+
+
+'''Method Interface'''
+
+<pre>
+function verifyMessage({
+  message: string,
+  data: string,
+  publicKey: string,
+}): Promise<{result: boolean}>
+</pre>
+
+
+'''Input arguments'''
+
+<pre>
+interface VerifyMessageArgs {
+  message: string; // Salt prefix + original message
+  data: string; // Signed message
```

Comment:
perhaps the param name should be `signature`

Reply:
this makes sense. Will update.